src/components/Company/Offers/Manage/CompanyOffersManagementWidget.spec.js

```diff
@@ -21,7 +24,7 @@ describe("App", () => {
             description: "kek",
         },
         {
-            id: "random uuid5",
+            _id: "random uuid5",
             title: "Guy in the background",
             ownerName: "Reddit",
             ownerLogo: "logo.com",
```

Comment:
The mock offers should have an `owner` property referring to the id of the company.